releasenotes/notes/0.46/prepare-0.46.0-c3857463a3c052f7.yaml

```
@@ -0,0 +1,620 @@
+    use the :func:`~.transpile` function (with appropriate transpile
+    options) followed by ``backend.run()`` (with appropriate run options).
+
+    For example, instead of running::
+
+      from qiskit import execute
+      job = execute(circuit, backend)
+
+    you can run::
+
+      from qiskit import transpile
+      new_circuit = transpile(circuit, backend)
+      job = backend.run(new_circuit)
+
+    Alternatively, the ``Sampler`` primitive is semantically equivalent to the
+    deprecated :func:`~.execute_function.execute` function. The class
+    :class:`.BackendSampler` is a generic wrapper for backends that do not support
+    primitives:
+
+      from qiskit.primitives import BackendSampler
+      sampler = BackendSampler(backend)
+      job = sampler.run(circuit)
+
+  - |
+    Implicit conversion from a dense ``BaseOperator`` to a :class:`~.SparsePauliOp`
+    in ``Estimator`` observable arguments is deprecated as of Qiskit 0.46 and will
+    be removed in Qiskit 1.0. You should explicitly convert to a SparsePauli op
+    using ``SparsePauliOp.from_operator(op)`` instead.
+
+  - |
+    The discrete pulse library is now deprecated and will be removed in a future release. This includes:
+
+    * :func:`~qiskit.pulse.library.constant`
+    * :func:`~qiskit.pulse.library.zero`
+    * :func:`~qiskit.pulse.library.square`
+    * :func:`~qiskit.pulse.library.sawtooth`
+    * :func:`~qiskit.pulse.library.triangle`
+    * :func:`~qiskit.pulse.library.cos`
+    * :func:`~qiskit.pulse.library.sin`
+    * :func:`~qiskit.pulse.library.gaussian`
+    * :func:`~qiskit.pulse.library.gaussian_deriv`
+    * :func:`~qiskit.pulse.library.sech`
+    * :func:`~qiskit.pulse.library.sech_deriv`
+    * :func:`~qiskit.pulse.library.gaussian_square`
+    * :func:`~qiskit.pulse.library.drag`
+
+    Instead, use the corresponding :class:`~qiskit.pulse.SymbolicPulse`, with :meth:`~.SymbolicPulse.get_waveform()`.
+    For example, instead of ``pulse.gaussian(100,0.5,10)`` use ``pulse.Gaussian(100,0.5,10).get_waveform()``.
+
+    Note that the phase of both ``Sawtooth`` and ``Square`` is defined such that a phase of :math:``2\\pi``
+    shifts by a full cycle, contrary to the discrete counterpart. Also note that complex amplitude support is
+    deprecated in the symbolic pulse library - use ``float`` ``amp`` and  ``angle`` instead.
+
+  - |
+    Deprecated the :class:`.ConfigurableFakeBackend` class, which has mainly been used
+    for internal testing. It will be removed in the Qiskit 1.0.0 release. Instead, you
+    can use the :class:`.GenericBackendV2` class to build a similar backend for testing.
+
+  - |
+    Loading library :class:`~.qiskit.pulse.ScalableSymbolicPulse` objects with
+    complex ``amp`` parameter from
+    qpy files of version 5 or lower (Qiskit Terra < 0.23.0) is now deprecated.
+    Following the removal in Qiskit 1.0.0, complex ``amp`` will be automatically
+    converted to float (``amp``,``angle``). The change applies to the pulses:
```

the backticks in angle are not rendering properly, this might help:

```suggestion
    converted to float (``amp``, ``angle``). The change applies to the pulses:
```